schemas/HPXMLDataTypes.xsd

```diff
@@ -2172,6 +2181,12 @@
 			<xs:enumeration value="other exterior"/>
 			<xs:enumeration value="roof deck"/>
 			<xs:enumeration value="unconditioned space"/>
+			<xs:enumeration value="other heated space"/>
```

Comment:
UnitLocation type is used in `WaterHeatingSystem/Location`, `HVAC/HVACPlant/HeatingSystem | CoolingSystem | HeatPump/UnitLocation`, and `Project/ProjectDetails/Measures/Measure/Location`. I think for these usage, it's probably ok to directly add new enumerations. Does it make sense?

Reply:
That seems reasonable.